Wrapping/R/CMakeLists.txt

```diff
@@ -91,6 +91,33 @@ if(NOT SimpleITK_FORBID_DOWNLOADS)
     )
 
 endif()
+# Download tar of Rd files
+if(NOT SimpleITK_FORBID_DOWNLOADS)
+  include(sitkExternalData)
```

Comment:
This is duplication from above, I think the code block below and above should be consolidated, with just one SimpleITKRData target.